multisite/template/loaders/filesystem.py

```diff
@@ -6,15 +6,20 @@
 from django.conf import settings
 from django.contrib.sites.models import Site
 from django.template.loaders.filesystem import Loader as FilesystemLoader
+from django import VERSION as django_version
 
 
 class Loader(FilesystemLoader):
-    def get_template_sources(self, template_name, template_dirs=None):
+    def get_template_sources(self, *args, **kwargs):
+        template_name = args[0]
         domain = Site.objects.get_current().domain
         default_dir = getattr(settings, 'MULTISITE_DEFAULT_TEMPLATE_DIR',
                                         'default')
         for tname in (os.path.join(domain, template_name),
                       os.path.join(default_dir, template_name)):
-            for item in super(Loader, self).get_template_sources(tname,
-                                                         template_dirs):
+            if django_version < (2, 0, 0):
+                args = [tname, None]
```

Comment:
For these older django versions, the default of the `template_dirs` kwarg is `None` anyway in the parent's `get_template_sources` right? So if we're always just passing `None` here anyway, do we need to bother checking the django version?

Reply:
Yes, because it was an arg in django 1.7 and a kwarg with =None in later django and now it's gone altogether.  Tests break under 1.7 if we don't give it the explicit args.

Reply:
Ah, okay. That's fine then